proposals/wac-ucr/index.bs

```diff
@@ -1114,13 +1114,178 @@ and the presentation goes off perfectly.
 Requirements {#requirements}
 ================================================================================
 
-Issue: Populate requirements based on established use cases.
+## Functional Requirements ## {#req-functional}
+
+1. The system shall permit or restrict an [=agent's=] ability to access a 
+    [=resource=] or [=collection=]. [[#uc-basic]], 
+    [[#uc-collections]], [[#uc-blockagents]]
+
+1. The system shall allow access to be limited based on the [=identity=] of 
+    the [=agent=]. [[#uc-basic]], [[#uc-collections]]
+
+1. The system shall allow access to be limited to an [=agent=] 
+    based on the [=agent's=] membership in a certain group of [=agents=]. 
+    [[#basic-group]]
+    
+1. The system shall allow access to be limited to an [=agent=] 
+    based on the client [=application=] in use by the [=agent=]. 
+    [[#uc-trusted-applications]]
+
+1. The system shall allow for access to a certain resource to be limited to 
+    the [=agent=] that created that resource.
+    [[#collection-readappendwrite]]
+
+1. The system shall allow access to be limited to an [=agent=] 
+    based on the [=agent's=] possession of a certain verifiable credential
+    or capability. FUTURE. 
+    [[#capabilities-vc]], [[#capabilities-link]], [[#uc-minimalcredentials]]
+    
+1. The system shall allow access to be limited to any 
+    unauthenticated [=agent=].
+    [[#basic-public]]
```

Comment:
What's "allowed"? Including write or control access? Wiping off the whole storage?

As I read "limited to", it seems like access is only given to unauthenticated agents as opposed to in addition to authenticated agents.

The UC description is about read access and if loosely being sharable - perhaps entailing that others can also "read".

Reply:
Clarified in https://github.com/solid/authorization-panel/pull/152/commits/65102d485f3e1e8af622e2c5f8edd70c01867250